.github/workflows/ci.yaml

```diff
@@ -71,6 +71,9 @@ jobs:
             - name: Run Integ Tests
               run: npm run integ:local:${{ matrix.browser }}:headless
```

Comment:
We can get rid of `browser` from the test matrix, since we're not using it.

Running the integ tests on each browser sequentially within a job is a bit more efficient anyway.